site2/docs/client-libraries-python.md

```diff
@@ -288,3 +288,103 @@ class Example(Record):
     a = String()
     sub = MySubRecord()
 ```
+
+## End-to-end encryption
+
+[End-to-end encryption](https://pulsar.apache.org/docs/en/next/cookbooks-encryption/#docsNav) allows applications to encrypt messages at producers and decrypt at consumers.
+
+### Configuration
+
+If you want to use the end-to-end encryption feature in the Python client, you need to configure `publicKeyPath` and `privateKeyPath` for both producer and consumer.
+
+```
+publicKeyPath: "./public.pem"
+privateKeyPath: "./private.pem"
+```
+
+### Tutorial
+
+This section provides step-by-step instructions on how to use the end-to-end encryption feature in the Python client.
+
+#### Prerequisite
```

Comment:
Heading is used to make the structure clear, in this case, I don't think we need a heading 4 for the prerequisite. I suggest that we use a "bold font" for prerequisite or a sentence for explain it.